frontend/src/stores/MainStore.js

```diff
@@ -206,6 +206,34 @@ class MainStore {
     @computed get getDatasetTypeName() {
         return modelTypes.find(item => item.value == this.dataset_type);
     }
+
+    @computed get isModelSelected() {
```

Comment:
Update names to be more descriptive; `isModelSelected` to `hasAtLeastOneModelSelected`.